### DISCUSSION TITLE
SVT subtitles

### DISCUSSION THREAD

**Discussion by w23sh** [stefansundin/privatkopiera#192] [Q&A]
No subtitles when I download from SVT.se,it worked in the past.

**Comment by stefansundin** [comment]
Hello. Can you please link a few videos with this issue? Thanks.

**Reply by docomp** [reply]
https://www.svtplay.se/video/jbmXw6z/vetenskapens-varld-manens-baksida?position=338&id=jbmXw6z
och
https://www.svtplay.se/video/8q1Epvz/allt-for-sverige/avsnitt-1?position=211&id=8q1Epvz
Båda publicerade nyligen 15-16/10.

Testade https://www.svtplay.se/video/jMVnb39/dromslottet/1-cider--och-vinagerrummet?id=jMVnb39  som publicerades för länge sedan och den fungerade fortfarande.

**Reply by daspunner** [reply]
I also have this issue. 
All episodes of these series:

https://www.svtplay.se/das-boot

and 

https://www.svtplay.se/babylon-berlin

/D

**Comment by Bear939** [comment]
Hi Stefan. Downloaded an episode of Das Boot from Svt play but can't get any subtitle. Can it be fixed?
Thanks for a really good program.

**Comment by ffsthis** [comment]
I use a workaround I download the text file and tell VLC to use it.
I hope the extension can be fixed to work again.

**Reply by daspunner** [reply]
How do you download the textfile?

**Reply by ffsthis** [reply]
It's the last file in the list underneath the mp4 files of various sizes.
If you get a zero kb empty file fix it by using a version of ffmpeg.exe that works,
Try this one for windows 32 bit. 
EDIT.
https://github.com/yt-dlp/FFmpeg-Builds/releases/download/autobuild-2023-12-17-14-08/ffmpeg-n6.1-6-g884a660cae-win32-gpl-6.1.zip

**Reply by ffsthis** [reply]
The subtitle file is listed as - "text-open.vtt".

**Reply by daspunner** [reply]
Getting empty files even with the version of FFMPEG that you provided a link to. Any ideas?

**Reply by lemonelv** [reply]
Also getting empty files even with the linked ffmpeg version, and the subtitle file is listed as "text-closed.vtt" for me. 
![image](https://github.com/stefansundin/privatkopiera/assets/155365640/3e877726-8d4f-4a1e-a1cd-bad0f2d319a4)


I'm trying to download the subtitles for julkalendern https://www.svtplay.se/julkalendern-trolltider-legenden-om-bergatrollet

**Reply by ffsthis** [reply]
Working even with the text-closed.vtt.
Julkalendern- Trolltider - legenden om bergatrollet - Avsnitt 24.srt  size is 6 KB 
Maybe you didn't succeed in replacing your old ffmpeg.exe with the new one I linked to.
I will download it again from that link and check to see if it is exactly the one I'm using.
I'm re-downloading it and will do a sha1 hash check on it.

**Reply by ffsthis** [reply]
I confirm the file in my link has changed since I downloaded it, and no longer works.
It is downloading an empty file.
Don't blame me, the one I have saved works fine from exactly the same link but earlier..

**Reply by ffsthis** [reply]
My good file is dated 16th Dec and is 96.5 MB (101,187,584 bytes) zipped
So maybe download one with that date and size .

**Reply by ffsthis** [reply]
Not the same file as mine but working.
ffmpeg-n6.1-6-g884a660cae-win32-gpl-6.1.zip
https://github.com/yt-dlp/FFmpeg-Builds/releases/download/autobuild-2023-12-17-14-08/ffmpeg-n6.1-6-g884a660cae-win32-gpl-6.1.zip
This file works

**Reply by lemonelv** [reply]
Ahh thank you so much, it works!! I was having trouble finding it so I appreciate it :))

**Reply by ffsthis** [reply]
Good.

**Comment by Lunarwalker60** [comment]
Copy text-link and change 'closed' to 'open'. Works for me.

**Reply by ffsthis** [reply]
h ttps://svt-vod-9j.akamaized.net/d0/se/20240101/6f46e9db-7b4f-4b63-99f7-9abc2afc
4cd9/text/text-open.vtt: Server returned 404 Not Found

So it didn't work by changing closed to open.

**Reply by Lunarwalker60** [reply]
My bad! I tried 'https://svt-vod-9j.akamaized.net/d0/se/20240101/6f46e9db-7b4f-4b63-99f7-9abc2afc4cd9/text/text-closed.vtt' in addressfield of Chrome and it worked so maybe it IS an error in ffmpeg.

**Reply by Lunarwalker60** [reply]
I checked how I normally do it and I dont use ffmpeg for SVT subtitles at all. I download the .vtt with curl and convert it in my own project in VB6. But if you want to do a conversion to .srt I recommend SubtitleEdit.

**Reply by drallegri** [reply]
Hi, got it working with help from this thead. Worked for SVTPlay as well. Just find the .vtt file, download it and voila, you're home.
https://github.com/stefansundin/privatkopiera/discussions/163#discussioncomment-8288759

**Reply by Lunarwalker60** [reply]
För de som eventuellt har problem med undertexter till SVT pga ffmpeg-version o dylikt har jag gjort en makroaktiverad Excel i Office 2016 som kan ta emot det kopierade kommandot från Privatkopiera och sen ladda ner vtt-filen och spara den som en srt. Det finns även ett Exe-program som gör samma sak plus kan konvertera en befintlig vtt till srt om man föredrar det men exe-filen kan kräva en del stödbibliotek som alla inte har installerade. Excel-versionen bör dock fungera om Du har Office installerat.
[SVT Undertext.zip](https://github.com/stefansundin/privatkopiera/files/14575447/SVT.Undertext.zip)
